linker/Mono.Linker.Steps/ResolveFromXmlStep.cs

```diff
@@ -59,11 +59,15 @@ public class ResolveFromXmlStep : ResolveStep {
 
 		XPathDocument _document;
 		string _xmlDocumentLocation;
+		string _resourceName;
+		AssemblyDefinition _resourceAssembly;
 
-		public ResolveFromXmlStep (XPathDocument document, string xmlDocumentLocation = "<unspecified>")
+		public ResolveFromXmlStep (XPathDocument document, string xmlDocumentLocation = "<unspecified>", string resourceName = null, AssemblyDefinition resourceAssembly = null)
```

Comment:
This should be different ctor as passing null to only one of the new values does not make sense

Reply:
Done